shim.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var define = require('define-properties');
+var getPolyfill = require('./polyfill');
+
+module.exports = function shimArrayPrototypeFind() {
+	var polyfill = getPolyfill();
+	if (Array.prototype.find !== polyfill) {
+		define(Array.prototype, { find: polyfill });
```

Comment:
`define` won't override existing properties unless there's a corresponding "should override" function - ie, here you want `define(Array.prototype, { find: polyfill }, { find: function () { return true; })`. Alternatively you could move the "if" condition into the predicate function.